### PR TITLE
Add Built In to INTHEWILD.md

### PR DESCRIPTION
**Pull request by beeceej**
Adds Built In to INTHEWILD.md

Thanks for this awesome tool!

### CODE REVIEW

**Comment by john-bodley** [comment]
@beeceej would you mind rebasing? https://github.com/airbnb/knowledge-repo/pull/537 should have resolved the CI issues.

**Comment by beeceej** [comment]
> @beeceej would you mind rebasing? #537 should have resolved the CI issues.

No problem, will do.

File: INTHEWILD.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ to add your organization and/or project to this document!
 Organizations
 ----------
  - [Airbnb](https://github.com/airbnb)
+ - [Built In](https://github.com/builtinx)
```
Comment (tomislavmitic2012) [inline INTHEWILD.md]: do we want to be included in this? should we ask management first before we do this.

Comment (beeceej) [reply INTHEWILD.md]: > do we want to be included in this? should we ask management first before we do this.

Yes, this has been approved already.

**Pull request merged**